Error running install_normaliz_with_eantic.sh

For me, running `install_normaliz_with_eantic.sh` dies in an error like this (I assume other `install_normaliz*.sh` variants are affected as well, but I didn't try):

```
...
OPT_LIB_DIR

installing with --disable-shared
Found see next line
true
Hiding Linux
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
configure: error: source directory already configured; run "make distclean" there first
```

This leaves things in a broken state, as shared libraries were moved to `local/lib/save_dynamic/` and remain there.

--- a/INSTALL
+++ b/INSTALL
@@ -36,7 +36,6 @@ doc/Normaliz.pdf, Section 11.2.
 They installation can in addition be controlled by several environment variables:
 
 NMZ_COMPILER: overrides the standard choice of compiler (g++ on Linux, clang++ on Mac)
-NMZ_SHARED: if set, only shared libraries will be built; binaries depend on all of them.
 NMZ_PREFIX: if set defines the path for make install; 
     by default it is "local" in the Normaliz directory
 GMP_INSTALLDIR: sets the path to the GMP files
@@ -51,19 +50,6 @@ EXTRA_FLAGS: One can further control the build process of Normaliz by setting op
    ./configure --help
 
 Have a look at the install scripts.
-
-*****************************
-****   TROUBLESHOOTING   ****
-*****************************
-
-In order to produce the semistatic binary (Linux and MacOS) install_normaliz.sh must make
-the dynamic libraries invisble: after copying them to the subdirectory save_dynamic, 
-removes them /local/lib and restores them after the build.
-
-If you should encounter any problems at this point, manual intervention is required.
-Since the dynamic libraries have been copied to /local/lib/save_dynamic before
-deletion, you can copy them back to /local/lib. If the worst comes to the worst, you
-must re-install the optional packages.
 
 
 ********************

--- a/doc/Normaliz.tex
+++ b/doc/Normaliz.tex
@@ -6576,12 +6576,6 @@ The binary \verb|normaliz| is stored in \verb|local/bin|, but it is also copied 
 
 Remarks:
 \begin{arab}
-\item By
-\begin{Verbatim}
-export NMZ_SHARED=yes  
-./install_normaliz_...
-\end{Verbatim}
-one can build Normaliz completely with shared libraries.
 \item It is possible to compile a statically linked Linux version of  \verb|normaliz|  through the \verb|Makefile.classic| in the directory \verb|source| after the installation of the optional libraries.
 
 \item If you want a global installation (and have the rights to do it), you can ask for

--- a/install_normaliz.sh
+++ b/install_normaliz.sh
@@ -47,70 +47,6 @@ if [ "x$NO_OPENMP" != x ]; then
     export BLOCK_OPENMP="--disable-openmp"
 fi
 
-if [ "x$NMZ_SHARED" = x ]; then
-
-    echo "installing with --disable-shared"
-    
-    mkdir -p ${OPTLIBDIR}/save_dynamic
-    
-    if [[ $OSTYPE == darwin* ]]; then
-        for file in ${OPTLIBDIR}/*.dylib*; do
-            if [[ -f $file ]]; then
-                FOUND=true
-                break
-            fi
-        done
-        echo "Found see next line"
-        echo $FOUND
-        if [ x$FOUND != x ]; then
-            echo "Hiding Mac"
-            cp -f ${OPTLIBDIR}/*.la* ${OPTLIBDIR}/save_dynamic
-            cp -f ${OPTLIBDIR}/*.dylib* ${OPTLIBDIR}/save_dynamic
-            rm -f ${OPTLIBDIR}/*.dylib*
-            rm -f ${OPTLIBDIR}/*la
-        fi
-    else
-        for file in ${OPTLIBDIR}/*.so*; do
-            if [[ -f $file ]]; then
-                FOUND=true
-                break
-            fi
-        done
-        echo "Found see next line"
-        echo $FOUND
-        if [ x$FOUND != x ]; then
-            echo "Hiding Linux"
-            cp -f ${OPTLIBDIR}/*.so* ${OPTLIBDIR}/save_dynamic
-            cp -f ${OPTLIBDIR}/*.la* ${OPTLIBDIR}/save_dynamic
-            rm -f ${OPTLIBDIR}/*.so*
-            rm -f ${OPTLIBDIR}/*la
-        fi
-    fi
-
-    
-    mkdir -p build_static
-    
-    cd build_static
-    
-    ../configure --prefix="${PREFIX}" --with-cocoalib="${PREFIX}" --with-flint="${PREFIX}" $EXTRA_FLAGS $WITH_GMP --disable-shared ${BLOCK_OPENMP} --srcdir=..
-
-    make clean
-    make -j4
-    make install
-    
-    #make distclean
-    
-    cd ..
-    #rm -r build_static
-    
-    ## we copy so and la back to their proper location
-
-    cp -f ${OPTLIBDIR}/save_dynamic/* ${OPTLIBDIR}
-    
-    mkdir -p ${PREFIX}/bin/hide ## hide the non-shared built binaries
-    mv -f ${PREFIX}/bin/*no* ${PREFIX}/bin/hide
-fi
-
 echo "installing shared"
 
 mkdir -p build_shared
@@ -126,11 +62,6 @@ make install
 cd ..
 # rm -r build_shared
 
-
-if [ "x$NMZ_SHARED" = x ]; then
-    mv -f ${PREFIX}/bin/hide/* ${PREFIX}/bin
-    rmdir ${PREFIX}/bin/hide
-fi
 
 if [[ $OSTYPE == darwin* ]]; then
         if [ "x$NMZ_MAC_STATIC" != x ]; then

